### ISSUE TITLE
Proxy-Error while using ember-cli

### ISSUE THREAD

**Issue by weblogixx** [ember-cli/ember-cli#3473]
Hi all,

I am getting the following error when trying to use ember-cli:

```
ember
version: 0.2.0
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: connect ECONNREFUSED
    at exports._errnoException (util.js:746:11)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:983:19)
```

I am using the following setup:
```
Windows 7 (64 bit)
node 0.12.0
npm 2.5.1
```
Correct proxy settings (they are working with other packeges and i can install and update packages via npm, so this does not seem like the problem).

I just looked at the problem and it seems like it is caused by the get-request that takes place in UpdateChecker.prototype.checkNPM. When uncommenting it, ember cli seems to run fine.

This is the error that is thrown:

```js
// This is my own test, just inserted above the promise
  http.get('https://registry.npmjs.org/ember-cli/latest', function(res) {
      console.log(res);
  }).on('error', function() {
      console.log(arguments);
      throw 'defect!';
  });
```

This gives me the following result:

```
ember
version: 0.2.0
{ '0':
   { [Error: connect ECONNREFUSED]
     code: 'ECONNREFUSED',
     errno: 'ECONNREFUSED',
     syscall: 'connect' } }

USERPATH\AppData\Roaming\npm\node_modules\ember-cli\lib\models\update-checker.js:75
      throw 'defect!';
      ^
defect!
```

These are my current npm settings (user/hostnames changed to examples):
```
npm config list
; cli configs
user-agent = "npm/2.5.1 node/v0.12.0 win32 x64"

; userconfig USERDIR\.npmrc
http-proxy = "http://proxy.company:8080"
https-proxy = "http://proxy.company:8080/"

; builtin config undefined
prefix = "USERDIR\\AppData\\Roaming\\npm"
```

Is there anything else I could do? It seems to be running fine some months ago with ember-cli 0.1.5.

**Comment by ziggurat** [comment]
I have the same issue. npm and bower both have the proxy configured and works fine.

I am on 
* Ubuntu 64bits, 
* node 0.12.0
* npm 2.5.1

**Comment by xomaczar** [comment]
this should be fixed in #3488
